src/components/footer/footer.module.scss

```diff
@@ -1,5 +1,5 @@
 .infoRepo {
-  font-weight: 100;
+  font-weight: 400;
```

Comment:
Could you add a before and after photo of it?

Reply:
Sure.